packages/docusaurus-preset-bootstrap/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "docusaurus-preset-bootstrap",
```

Comment:
`@docusaurus/preset-bootstrap`. We need to prefix our internal packages with the `@docusaurus` scope